Usage.md

```diff
@@ -214,12 +232,90 @@ litmusctl get agents --project-id=""
 **Output:**
 
 ```
-AGENTID                                AGENTNAME          STATUS 
-55ecc7f2-2754-43aa-8e12-6903e4c6183a   agent-1            ACTIVE 
-13dsf3d1-5324-54af-4g23-5331g5v2364f   agent-2            INACTIVE
+AGENTID                                AGENTNAME          STATUS     REGISTRATION
+55ecc7f2-2754-43aa-8e12-6903e4c6183a   agent-1            ACTIVE     REGISTERED
+13dsf3d1-5324-54af-4g23-5331g5v2364f   agent-2            INACTIVE   NOT REGISTERED
+```
+
+
+* To disconnect an agent, issue the following command..
+```shell
+litmusctl disconnect agent f9799723-29f1-454c-b830-ae8ba7ee4c30 --project-id=""
```

Comment:
Can you also add one more command displaying agent_id as a placeholder instead of the actual agent_id and use the current command as an example.
`litmusctl disconnect agent <agent_id> --project-id="<project_id>"`